values.yaml

```diff
@@ -226,6 +183,87 @@ graphdb:
       initialDelaySeconds: 60
       timeoutSeconds: 5
       periodSeconds: 10
+    # additional environment variables to be set for the graphdb nodes
+    extraEnvFrom: []
+    # additional volumes to be set for the graphdb nodes
+    extraVolumes: []
+    # additional volume mounts to be set for the graphdb nodes
+    extraVolumeMounts: []
+    # securityContext defines privilege and access control settings for the node pods.
+    securityContext: {}
+    # securityContext defines privilege and access control settings for the node containers (including provisioning ones).
```

Comment:
securityContext -> containerSecurityContext

Reply:
No, follow `securityContext` and `podSecurityContext`

Reply:
i was talking about the comment, it's fixed now